sample-compose/app/build.gradle.kts

```diff
@@ -30,13 +30,14 @@ android {
         }
     }
 
-    compileSdk = Versions.ANDROID_COMPILE_SDK_VERSION
+    namespace = "co.nimblehq.sample.compose"
+    compileSdk = libs.versions.androidCompileSdk.get().toInt()
     defaultConfig {
         applicationId = "co.nimblehq.sample.compose"
-        minSdk = Versions.ANDROID_MIN_SDK_VERSION
-        targetSdk = Versions.ANDROID_TARGET_SDK_VERSION
-        versionCode = Versions.ANDROID_VERSION_CODE
-        versionName = Versions.ANDROID_VERSION_NAME
```

Comment:
We need to migrate its reference in workflows
- https://github.com/nimblehq/android-templates/blob/692a83c615fc69e027aac4a89a7aacdb3a4e667c/.github/workflows/bump_version.yml#L29
- https://github.com/nimblehq/android-templates/blob/692a83c615fc69e027aac4a89a7aacdb3a4e667c/.cicdtemplate/.github/workflows/bump_version.yml#L25
- https://github.com/nimblehq/android-templates/blob/692a83c615fc69e027aac4a89a7aacdb3a4e667c/.cicdtemplate/.github/workflows/create_release_pull_request.yml#L19